src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public class IdentityLogger
```

Comment:
They might use it in a scenario where they want to change the implementation of how the IdentityResult, SignInResult, bool etc are logged. I am assuming that would be a very small subset of users. I can move it to a different namespace that's all right ? I don't think we should make the class sealed, remove virtual off methods or make it internal.